stock/templates/registered/operationmanagement.html

```diff
@@ -0,0 +1,70 @@
+{% extends "public/base.html" %}
+{% load staticfiles %}
+
+{% block title %}Alliance Auth{% endblock %}
+
+{% block page_title %}Fleet Operation Management{% endblock page_title %}
+{% block extra_css %}{% endblock extra_css %}
+
+{% block content %}
+    <div class="col-lg-12">
+        <h1 class="page-header text-center">Fleet Operation Timers
+            <div class="text-right">
+                <a href="{% url 'auth_add_optimer_view' %}">
+                    {% if perms.auth.optimer_management %}
+                        <button type="button" class="btn btn-success">Create Operation</button>
+                    {% endif %}
+                </a>
+            </div>
+        </h1>
+        <table class="table table-responsive table-bordered">
+            <tr>
+                <th class="text-center">Operation Name</th>
+                <th class="text-center">Form Up System</th>
+                <th class="text-center">Form Up Location</th>
+                <th class="text-center">Start Time</th>
+		<th class="text-center">End Time</th>
+                <th class="text-center">FC</th>
+		<th class="text-center">Details</th>
+		<th class="text-center">Post Time</th>
+		{% if perms.auth.optimer_management %}
+           <th class="text-center">Creator</th> 
+		   <th class="text-center">Action</th>
+                {% endif %}
+            </tr>
+            	              
+                {% for ops in optimer %}
+                <tr>
+       		<td style="width:150px" class="text-center">{{ ops.operation_name }}</td>
```

Comment:
yes it does because html when it renders doesn't read mixed spaces. it will only read them if you put breaks in them.